### ISSUE TITLE
[BUG] Meteor Lake w/Intel ARC Installer Freezes on Boot Mode Selection

### ISSUE THREAD

**Issue by xags** [LibreELEC/LibreELEC.tv#9005]
### Describe the bug
When using a properly created USB installer image, the USB stick boots to the install screen waiting to select a boot mode (installer,live,run). Once selecting any of the boot modes the system freezes.

### How to reproduce
1. Boot via USB (UEFI or Legacy) (using standard AMD64 or Legacy Xorg image)
2. Install selection screen shows
3. Select either installer, live, or run.
4. Machine freezes, USB appears to disconnect both the keyboard and USB stick.

### Information
 - LibreELEC Version: 12.0 and latest 6/19/24 13.0 nightly
 - Hardware Platform: Asus NUC Pro ( NUC14RVK with Intel Core Ultra 5 125H Processor)

### Log file
No messages are logged by the installer itself.

### Context
Unable to get the installer to even start.

**Comment by xags** [comment]
I've also tried modifying the syslinux.cfg on the USB stick to remove the quiet option. The only additional output is: "Loading kernel: OK"

**Comment by smp79** [comment]
I think CONFIG_PINCTRL_METEORLAKE needs to be enabled in kernel config.

**Comment by smp79** [comment]
Could you try [this image](https://www.dropbox.com/scl/fi/g35qlpbv8lgnskpiqr4h3/LibreELEC-Generic.x86_64-13.0-devel-20240620031939-fb39b80.img.gz?rlkey=i87qbod97f2b4b217x7ugpkcx&st=euoi2wjz&dl=1)?

**Comment by xags** [comment]
I'm not sure what I did but the 6/19/24 nightly started working after I disabled VT-D/IVT/TPM and maybe CEC. What's frustrating is that when I re-enable them the 13.0 nightly still boots properly!  If I default the bios to "optimal defaults" it fails until I repeat the enable/disable procedure. Guessing a BIOS bug?

Your image doesn't cause the machine to lock-up but it gives an error: Error in mount_flash: mount_common: Could not mount UUID=2006-0908

The official 12.0 image does the same thing as your image with a different UUID in the error message. I've dd'd the image multiple times and re-downloaded to ensure it wasn't corrupt.
